src/NITTA/Utils.hs

```diff
@@ -122,15 +122,12 @@ endpointAt t p =
 getCad Step{pDesc} | CADStep cad <- descent pDesc = Just cad
 getCad _ = Nothing
 
-isFB s = isJust $ getFB s
+isIntermediate s = isJust $ getIntermediate s
 
-getFB Step{pDesc} | FStep fb <- descent pDesc = Just fb
-getFB _ = Nothing
+getIntermediate Step{pDesc} | IntermediateStep f <- descent pDesc = Just f
+getIntermediate _ = Nothing
 
-getFBs p = mapMaybe getFB $ sortOn stepStart $ steps p
-
-getFunction Step{pDesc} | FStep role <- descent pDesc = Just role
-getFunction _ = Nothing
+getIntermediates p = mapMaybe getIntermediate $ sortOn stepStart $ steps p
```

Comment:
as I see, it is just renaming?

Reply:
Yes

Reply:
Maybe, you know any other places, that are forgotten by me.